lambdas/select-cri/src/main/java/uk/gov/di/ipv/core/selectcri/SelectCriHandler.java

```diff
@@ -144,7 +144,14 @@ private JourneyResponse getNextWebJourneyCri(
                         passportCriId,
                         passportCriId,
                         userId);
-        if (passportResponse.isPresent()) {
+        Optional<JourneyResponse> drivingLicenceResponse =
+                getCriResponse(
+                        visitedCredentialIssuers,
+                        currentVcStatuses,
+                        drivingLicenceCriId,
+                        drivingLicenceCriId,
+                        userId);
+        if (passportResponse.isPresent() && drivingLicenceResponse.isPresent()) {
             return passportResponse.get();
```

Comment:
Returning only passportResponse. Do we need to include drivingLicenceResponse in return too ?

Reply:
In this case they’d actually be the same, both send you to the doc check page which drives which of the two the user actually goes to. But correct me if I’m wrong @Danomanic

Reply:
It doesn't use the response (apart from checking it is present), so passing back either the passport or driving licence is fine here.